app/models/goal.py

```diff
@@ -3,3 +3,5 @@
 
 class Goal(db.Model):
```

Comment:
It would be good to have some helper methods here, like a method to convert a Goal to a dictionary.  Maybe also a class method that takes a dictionary and returns an instance of a Goal.